src/Freya.Integration/Integration.fs

```diff
@@ -22,6 +22,21 @@ module Freya.Integration
 open System
 open System.Threading.Tasks
 open Freya.Core
+open Freya.Pipeline
+
+(* OWIN Types *)
+
+/// Type alias of <see cref="FreyaEnvironment" /> in terms of OWIN.
+type OwinEnvironment =
+    FreyaEnvironment
+
+/// Type alias for the OWIN AppFunc signature.
+type OwinAppFunc = 
+    Func<OwinEnvironment, Task>
+
+/// Type alias for the OWIN MidFunc signature.
+type OwinMidFunc =
```

Comment:
I think this could make sense to move as it's not used actually used in Freya.Core. Still, this is a core concept that could benefit from being included.